scene/resources/audio_stream_polyphonic.h

```diff
@@ -54,7 +54,7 @@ class AudioStreamPolyphonic : public AudioStream {
 class AudioStreamPlaybackPolyphonic : public AudioStreamPlayback {
 	GDCLASS(AudioStreamPlaybackPolyphonic, AudioStreamPlayback)
 
-	enum {
+	enum : int64_t {
```

Comment:
Should it be unsigned if we're not using negative numbers?
```suggestion
	enum : uint64_t {
```

Reply:
Also relevant since we're using a mask and some compilers have problems with that for negative numbers as I recall

Reply:
> Should it be unsigned if we're not using negative numbers?

Makes sense to me, the enum values are mixed with or casted to signed ints in a few places but the intent seems to be for all of them to be unsigned.